.github/workflows/ci-e2e-badger.yaml

```diff
@@ -40,6 +40,14 @@ jobs:
             STORAGE=badger make jaeger-v2-storage-integration-test
             ;;
         esac
+          
+    - name: Upload Metrics Artifact
+      uses: actions/upload-artifact@v4
+      if: ${{ matrix.version }} == "v2"
+      with:
+            name: badger-metrics-${{ matrix.version }}
+            path: ./metrics/badger_metrics.txt
+            retention-days: 7
```

Comment:
Why do you keep switching between 7 and 60 days? I already explained that we need BOTH of those retention values depending on whether the job runs for a tagged release or not.

Reply:
how do we know if its a tagged release run or a normal run ?

Reply:
` ${{ github.event_name == 'release' && 60 || 7 }}` something like this ?

Reply:
no, we don't run these workflows on release event. When CI runs for a tag I think the `github.ref` variable has a value like `refs/tags/v1.0.0`, so we can check for that. 

But before we even get into all of that we need to define the main logic for comparing metrics artifacts between the current run and the last release. For example. there is this action https://github.com/benchmark-action/github-action-benchmark that does something similar but for benchmark results. It seems to suggest using actions/cache@v4 to store the results of the tagged release run.